pkg/util/kubernetes/hostinfo/no_apiserver.go

```diff
@@ -7,6 +7,6 @@
 
 package hostinfo
 
-func apiserverNodeLabels(nodeName string) (map[string]string, error) {
+func apiserverNodeLabels(nodeName string) (map[string]string, error) { //nolint
```

Comment:
Could you add a comment after the `//nolint` explaining why we don't lint it ?

Reply:
Yeah the method was never used, but I wanted to keep it for consistency with other no_*.go, but in fact I think it's useless, I removed the file thus removing the `//nolint` :D